src/commands/pack/win.ts

```diff
@@ -363,8 +363,12 @@ async function signWindows(
   o: string,
   arch: string,
   config: Interfaces.Config,
-  windows: Interfaces.PJSON.Plugin['windows'],
+  windows: Interfaces.OclifConfiguration['windows'],
 ) {
+  if (!windows) {
+    throw new Error('windows not set in package.json')
```

Comment:
This would be misleading if they are using an rc file. Maybe say "windows not set in oclif configuration" ??